linux/packaging/appimage/make_config.yaml

```diff
@@ -1,6 +1,6 @@
 display_name: Twake
 
-icon: assets/logo.png
+icon: assets/favicon.png
```

Comment:
fav must be the fav, logo must be the logo. Maybe fav is very small size

Reply:
I'll wait for rounded icon to add logo again